### ISSUE TITLE
Update to 6.2.2 Webpack not refreshing app after edits

### ISSUE THREAD

**Issue by mrwrighty** [NativeScript/nativescript-cli#5168]
Just upgraded to Nativescript 6.2.2.

tns debug iOS works fine, prompt says Webpack compilation complete. Watching for file changes.

Any edits to code results in Webpack compilation complete. Watching for file changes.
Webpack build done! but the app on the device (iOS Simulator) does not update. Everything worked fine on Nativescript 6.2.0

Have to kill CLI and rerun tns debug iOS for app to update.

**Comment by Fatme** [comment]
Can you update to the latest version of `nativescript-dev-webpack` and check if this will resolve the issue:
`npm i nativescript-dev-webpack && ./node_modules/.bin/update-ns-webpack --configs`

**Comment by mrwrighty** [comment]
OK have done as suggest, but now have a strange issue with my app.

I have a login screen of email address and password. Both are TextFields. I can click in to the password field and enter text but as soon as I click in to the email address field the whole app freezes and I can no longer type in either field. This worked fine before the update to Webpack

**Comment by mrwrighty** [comment]
Any update on this. I now have an unusable debug through the simulator.

**Comment by rosen-vladimirov** [comment]
Hey @mrwrighty ,
We'll need more information in order to investigate this issue. Can you please try the following:
1. Delete platforms and node_modules directories: `rm -rf platforms node_modules`.
2. Now try `tns debug ios` and after getting the message `Successfully synced application...`, try changing some file in your app and see what happens.
3. If the above does not work, can you please run `tns debug ios --log trace &> out.txt` and apply after application is restarted on Simulator, apply a change in the code. After several seconds, stop the command and send us the generated `out.txt` file.
4. What is your Xcode version: `xcodebuild -version` command should give you the output.
5. What is the version of the simulator you are using? 13.2.2 maybe?

**Comment by mrwrighty** [comment]
Hi Thanks for your prompt response.

Yes Simulator is 13.2.2. My iPhone X is 13.2.3.

XcodeBuild:- 

Xcode 11.2.1
Build version 11B500

Removed platforms and node_modules and rebuilt. Webpack says it has updated the files, but the changes are not reflected in the app on the simulator or my iPhone X. iPhone X app remains responsive on both textfields, but the simulator app locks out as soon as I click on the textfield to enter an email address. The password textfield remains active as long as I do not click the email textfield first.

out.txt file attached.

[out.txt](https://github.com/NativeScript/nativescript-cli/files/3910831/out.txt)

**Comment by rosen-vladimirov** [comment]
Hey @mrwrighty ,
Thanks a lot for sharing the logs. We've investigated them, but we were unable to get an idea what might cause the unexpected behavior. So, we've added some more trace messages in the `next` version of CLI, can you install it on your side `npm i -g nativescript@next` and execute `tns debug ios --log trace &> out.txt` after that (similar to the previous time - apply a change, wait 10-15 seconds and stop the process).
Thanks in advance for your help.

**Comment by mrwrighty** [comment]
OK update applied. out2.txt attached. Problem still exists.
[out2.txt](https://github.com/NativeScript/nativescript-cli/files/3911605/out2.txt)

**Comment by mrwrighty** [comment]
Further to this, I have created a new simple app using Sidekick to start a blank javascript app.

I have added a login TextField and a Password Textfield. Updating the pages seems to work and any changes to the designer are reflected in the page layout. However I still have the issue of the email Textfield locking out the app. The password field works fine. Once the app is locked out is fails to receive any changes to the code. I have attached a wetransfer link to an MOV files of a screen capture showing what is happening. https://wetransfer.com/downloads/ce0fd5b785b16d4d47bfa466430fb40b20191202143406/b75e21e3611e8af77c2314c1476ac0eb20191202143406/871f13

**Comment by rosen-vladimirov** [comment]
Hey @mrwrighty ,
Thanks for the prompt response. From the video it seems the app is not responding after you select the Email field. It seems the application has crashed or is just frozen. On my side, while trying to reproduce the issue, when I select the Email TextField, I see a keyboard shown on the Simulator.
Can you try reproducing the issue on other Simulator (select the Simulator and from the top menus - Hardware-> Device ->  iOS13.2 -> (Some device). After that try running the application on it.
If it still does not work, can you send us the application you've just created (the one you've shown in the video)?

**Comment by mrwrighty** [comment]
OK this is strange I have restarted the simulator (6s) and also started iPhone 8 simulator. Both textfields respond as expected however the design is still not being updated with changes in my original app.  Attached is the simple project shown in the video.

[testupdate.zip](https://github.com/NativeScript/nativescript-cli/files/3911890/testupdate.zip)

**Comment by rosen-vladimirov** [comment]
Hey @mrwrighty ,
Ok, seems like the issue with the new project is resolved. I've tested the sample you've sent and it is working correctly on my side. 
So, if we get back to the issue you've faced with your original project, if I understand correctly, once you select one of the TextFields in it, the application freezes (or at least it seems frozen) and no more changes are synced. Can you confirm that's the issue you are facing?
Also, can you try running `tns debug ios --log trace &> out.txt` and send us this log when you reproduce the issue (please use the `next` version of NativeScript CLI).

**Comment by mrwrighty** [comment]
Sorry for the delay in responding but got called on to another project. By restarting the device, the freezing seems to have stopped, however this particular project is still having issues updating the content after any changes. I will re-run the log trace and post shortly.

**Comment by mrwrighty** [comment]
OK log file created. 
[out.txt](https://github.com/NativeScript/nativescript-cli/files/3951347/out.txt)

**Comment by rosen-vladimirov** [comment]
Hey @mrwrighty ,
Sorry for the late response (long holiday for me 😄 ). I've investigated your logs and I found the following lines:
```
Found device with identifier 'f60a8450e535f436b76a56634eb020aff4c7c523'
Lost device with identifier 'f60a8450e535f436b76a56634eb020aff4c7c523'
Received deviceLost event in LiveSync service for f60a8450e535f436b76a56634eb020aff4c7c523. Will stop LiveSync operation for this device.
```
For some reason CLI decides the simulator/device is not reachable any more and that's why it does not sync changes to it anymore.

As its been a while since the time you've had this issue, do you still reproduce it on your side?

**Comment by mrwrighty** [comment]
Thanks for the update.
I appear to have fixed the issue. I had updated the Webpack.config.js file to ignore some SQLLite errors appearing on compile, although I did not have any issues compiling and running the app, it failed to update on any code changes. I cleared everything down such as platform, node_modules etc, updated everything to the latest versions which replaced the Webpack.config.js file and all is good again. Not sure if editing the Webpack.config.js file caused the issue, but it appears OK at the moment and changes are being updated.
